netbox/dcim/migrations/0051_merge_20171031_2024.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
```

Comment:
Where did this migration come from? It's empty and shoudn't be needed.

Reply:
This Migration was a requirement for the 49 migration you created at the same time that i created one. It merges your 49 and my 50. Is there a better way to do this?